android/src/main/java/com/swmansion/gesturehandler/core/HoverGestureHandler.kt

```diff
@@ -103,8 +105,13 @@ class HoverGestureHandler : GestureHandler<HoverGestureHandler>() {
         finish()
       }
 
+      this.state == STATE_ACTIVE && event.getToolType(0) == MotionEvent.TOOL_TYPE_STYLUS -> {
+        stylusData = GestureUtils.updateStylusData(event)
+      }
+
       this.state == STATE_UNDETERMINED &&
         (event.action == MotionEvent.ACTION_HOVER_MOVE || event.action == MotionEvent.ACTION_HOVER_ENTER) -> {
+        stylusData = StylusData()
```

Comment:
Shouldn't this also be inside `onReset`?

Reply:
It can be. Done in [ce3ebd1](https://github.com/software-mansion/react-native-gesture-handler/pull/3111/commits/ce3ebd1b83ba3535cf8303f231d3e6ffe48ac2d4).